lib/cortex/snippets/client/helper.rb

```diff
@@ -1,9 +1,11 @@
+require 'cortex/snippets/client/webpage'
+
 module Cortex
-  module Snippets
+  module Snippet
     module Client
       module Helper
-        def snippet(options = {}, &block)
-          snippets = webpage[:snippets] || []
+        def snippet(cortex_client, options = {}, &block)
+          snippets = current_webpage(cortex_client)[:snippets] || []
```

Comment:
I don't think this will return anything - it would either have to be `current_webpage(cortex_client).webpage[:snippets]`, or we'll have to provide a getter in `Webpage` for `snippets`. I believe the latter would be the ideal solution, since that's how we're accessing other pieces of the `webpage` - like SEO information, etc.